### ISSUE TITLE
Nextcloud: calDAV: false positive

### ISSUE THREAD

**Issue by YeapGuy** [coreruleset/coreruleset#3611]
### Description

An iPhone trying to PUT to Nextcloud's calDAV service will be blocked by CRS.

### Logs

```
[error] 97548#97548: *156197 [client xxx] ModSecurity: Access denied with code 403 (phase 2). Matched "Operator `Ge' with parameter `5' against variable `TX:BLOCKING_INBOUND_ANOMALY_SCORE' (Value: `5' ) [file "/usr/local/nginx/conf/conf.d/include/coreruleset/rules/REQUEST-949-BLOCKING-EVALUATION.conf"] [line "176"] [id "949110"] [rev ""] [msg "Inbound Anomaly Score Exceeded (Total Score: 5)"] [data ""] [severity "0"] [ver "OWASP_CRS/4.0.0"] [maturity "0"] [accuracy "0"] [tag "anomaly-evaluation"] [hostname "xxx"] [uri "/remote.php/caldav/calendars/xxx/xxx/xxx.ics"] [unique_id "171045623486.423063"] [ref ""], client: xxx, server: xxx, request: "PUT /remote.php/caldav/calendars/xxx/xxx/xxx.ics HTTP/1.1", host: "xxx"
```

### Your Environment

* CRS version (e.g., v3.3.4): 4.0.0
* Paranoia level setting (e.g. PL1) : PL1
* ModSecurity version (e.g., 2.9.6):
* Web Server and version or cloud provider / CDN (e.g., Apache httpd 2.4.54): Nginx-Proxy-Manager Plus latest
* Operating System and version: Dockerized

### Confirmation

[ ] I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

**Comment by EsadCetiner** [comment]
@YeapGuy Have you tried using the [Nextcloud rule exclusions plugin](https://github.com/coreruleset/nextcloud-rule-exclusions-plugin)? The plugin should cover most false positives that you encounter with Nextcloud.

**Comment by YeapGuy** [comment]
Yes, I am using that

**Comment by YeapGuy** [comment]
Oh, I just realised... it has its own repository 😄 Should I report there instead?

**Comment by EsadCetiner** [comment]
I can transfer the issue if you want, then I'll take a look at the issue in a few hours.
